packages/frontend/src/hooks/usePositions.ts

```diff
@@ -41,7 +42,28 @@ export const usePositions = () => {
     fetchPolicy: 'cache-and-network',
   })
 
-  useInterval(refetch, 30000)
+  useEffect(() => {
+    subscribeToNewPositions()
+  }, [])
+
+  const subscribeToNewPositions = useCallback(() => {
+    subscribeToMore({
+      document: SWAPS_SUBSCRIPTION,
+      variables: {
+        poolAddress: squeethPool?.toLowerCase(),
+        origin: address || '',
+        recipients: [shortHelper, address || '', swapRouter],
```

Comment:
this adjust the query to get swaps from short helper and the Uniswap swap router for individual sells after mints?

Reply:
Oh, this adjust is for subscription, I didn't change the swaps pulling logics

Reply:
Ok don't we need to adjust the swap query to make sure we get all Uniswap swaps not just ones from our short helper?

Reply:
but do we really miss some txs here?

Reply:
yeah if a user buys back on Uniswap directly it won't capture that right? Or if they just mint and then go sell

Reply:
Ah kk, maybe I  fix this issue in another PR?